polldaddy-shortcode.php

```diff
@@ -180,7 +180,7 @@ function polldaddy_shortcode( $atts ) {
 
 			$poll      = intval( $poll );
 			$poll_url  = sprintf( 'https://poll.fm/%d', $poll );
-			$poll_js   = sprintf( '%s.polldaddy.com/p/%d.js', '//static', $poll );
+			$poll_js   = sprintf( '%s.polldaddy.com/p/%d.js', '//secure', $poll );
```

Comment:
Which is this first placeholder even here? 

And why is the protocol-less anti-pattern used?